src/components/richLink.tsx

```diff
@@ -142,6 +142,21 @@ const richLinkStyles = (format: Format): SerializedStyles => {
 			);
 		}
 
+		${until.tablet} {
+			[data-font-size='110%'] &,
+			[data-font-size='120%'] &,
+			[data-font-size='130%'] &,
+			[data-font-size='140%'] &,
+			[data-font-size='150%'] &,
+			[data-font-size='160%'] & {
+				width: calc(100% - ${remSpace[3]} - ${remSpace[3]});
+				margin-right: 0;
+				.js-image {
+					display: none;
+				}
+			}
+		}
```

Comment:
Does this work for you?

```suggestion
		${until.mobileMedium} {
			width: 100%;
		}
```